### DISCUSSION TITLE
epkg-update gives `Please update the ‘epkg’ package.  The installed version is too old for the current database scheme.`

### DISCUSSION THREAD

**Discussion by pedz** [emacscollective/epkg#44] [Support] [answered]
The first time I do `epkg-update`, there is a long pause followed by the error (in the subject).  After the first time, the error returns immediately.  `borg-assimulate` also returns the error.  I'm on a Mac so I can delete my entire emacs lisp directory and restore it from Time Machine.  That's why I say "the first time".

I've not used emacs for about a year.  The epkg git repository is on the "master" branch with a time stamp of the last commit of October 2022.

I can't even find where this message is coming from.

My repository is `git@github.com:pedz/emacs.git`. I'm running Emacs 29.1.

**Comment by tarsius** [comment, marked as answer]
Update the package manually, using `cd lib/epkg; git pull; make clean lisp`. Do the same for borg. Then restart Emacs and try again.

**Comment by pedz** [comment]
You have changed from `master` to `main` so I did `cd lib/epkg; git fetch; git branch --track main origin/main; git checkout main` (that's from memory so it might be slightly off).  But now I get `No applicable method: emacsql-live-p, t`

So I did the same thing for `emacsql` but I still get the error.

**Reply by pedz** [reply]
I did the same thing for `closql` and now its working (so far).

Thank you for your help.  Spent almost the whole day trying to figure this out.

**Reply by tarsius** [reply]
Sorry about. Some time next year, I'll look into making borg ignore outdated epkg, after all it doesn't actually need epkg, it just can't cope with an outdated version.
